### DISCUSSION TITLE
Tabs | Ajuste de tamanho text-preset

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1278] [Proposals]
### Goal

Melhorar a  leiturabilidade da experiência da pessoa usuária e deixar o sistema mais consistente.

### Context

Nos fluxos do Novo App as abas ganham caráter de título em alguns contextos, precisamos ter um tamanho e peso de fonte equivalentes em outras páginas para garantir conforto na leitura das informações e consistência no sistema. Por esta razão, solicitamos o ajuste de text-preset 3 para text preset 4 (de: 16 -> para: 18).

### Impact

Aumentar o tamanho da tipografia no componente Tabs melhoraria a consistência no sistema e a experiência da pessoa usuária.

### Discussion Questions

Seria possível fazer esse ajuste na biblioteca Mística?

### Do you have a visual proposal?

https://www.figma.com/file/VkXWMSV45UmSpbwsiJ1svN/Tabs?type=design&node-id=1210%3A19304&t=ARPzIWPV9NTCCf3F-1

**Comment by yceballost** [comment]
Hola Anne! 

Puedes razonar en dónde este cambio hace que "sea más consistente"?

No consideráis que es un problema de jerarquía que tenga el mismo tamaño y peso visual que un title?
![image](https://github.com/Telefonica/mistica-design/assets/6722153/f1fbfd66-1179-4fea-bd8a-4b93e72e0f9c)

**Reply by annefranca** [reply]
@yceballost como mencionei acima, em alguns momentos, o componente assume a mesma propriedade de um título, segue o print.  
<img width="225" alt="Captura de Tela 2023-05-30 às 13 17 44" src="https://github.com/Telefonica/mistica-design/assets/127137234/08ccf5ed-0664-4833-8c59-986d1e005db7">

**Reply by annefranca** [reply]
@yceballost você tem alguma outra dúvida? Não sei o que você acha, mas poderíamos ter uma variante do componente Tabs SM e LG para atender às necessidades da Vivo sem alterar o tamanho da tipografia nas outras marcas.

**Reply by yceballost** [reply]
mmm esa es otra posibilidad que estuvimos valorando, pero justo por esa razón consideramos que no era suficiente para evolucionar el componente hacia esa dirección, así que finalmente se creará un token para que vivo pueda controlar el tamaño del texto de la tab

**Reply by annefranca** [reply]
Seria a criação de um component token?

**Reply by yceballost** [reply]
exacto

**Reply by annefranca** [reply]
@yceballost perfeito! Teria alguma relação com a discussão 1253 ou são cenários diferentes? https://github.com/Telefonica/mistica-design/discussions/1253

**Reply by yceballost** [reply]
son cosas diferentes. Aquí estamos hablando de tamaño, en el otro lado se discute sobre el peso de las tipografías.
